code/modules/server_tools/server_tools.dm

```diff
@@ -11,9 +11,12 @@ GLOBAL_PROTECT(reboot_mode)
 	ExportService("[SERVICE_REQUEST_IRC_BROADCAST] [msg]")
 
 /world/proc/ServiceEndProcess()
-	log_world("Sending shutdown request!");
-	sleep(1)	//flush the buffers
-	ExportService(SERVICE_REQUEST_KILL_PROCESS)
+	log_world("Self terminating!");
+	sleep(1)	//flush the chat buffers

```

Comment:
ew

Reply:
BYOND needs at least 1 tick to send queued outputs, this makes sure that happens before it dies

Reply:
give it 5

Reply:
no, do world.tick_lag, not 1. atm 1ds is two full ticks. byond only needs 1 tick.

or you could do a stop_lag() call. this will give it more ticks if there is alot going on.